src/@types/index.d.ts

```diff
@@ -0,0 +1,75 @@
+export {}
+
+declare global {
+  export enum VoteStates {
+    UPCOMING = 'UPCOMING',
+    IN_PROGRESS = 'IN_PROGRESS',
+    DONE = 'DONE',
+  }
+
+  export interface IAPIResponse {
+    apiVersion: string
+    knockout: Record<number, ISinglePoll>
+    groupStages: Record<number, ISinglePoll>
+    quarterFinal: Record<number, ISinglePoll>
+    semiFinal: Record<number, ISinglePoll>
+    runnerUp: ISinglePoll
+    final: ISinglePoll
+  }
+
+  export interface IGameData {
+    knockout: Record<number, import('../models/SinglePoll').default>
+    groupStages: Record<number, import('../models/SinglePoll').default>
+    quarterFinal: Record<number, import('../models/SinglePoll').default>
+    semiFinal: Record<number, import('../models/SinglePoll').default>
+    runnerUp: import('../models/SinglePoll').default
+    final: import('../models/SinglePoll').default
```

Comment:
Inline imports aren't considered best practise, have changed them out.
```suggestion
    knockout: Record<number, SinglePoll>
    groupStages: Record<number, SinglePoll>
    quarterFinal: Record<number, SinglePoll>
    semiFinal: Record<number, SinglePoll>
    runnerUp: SinglePoll
    final: SinglePoll
```